src/sentry/integrations/slack/utils.py

```diff
@@ -336,6 +336,19 @@ def strip_channel_name(name):
 
 
 def get_channel_id(organization, integration_id, name):
+
+    name = strip_channel_name(name)
+    try:
+        integration = Integration.objects.get(
+            provider="slack", organizations=organization, id=integration_id
+        )
+    except Integration.DoesNotExist:
+        return None
+
+    return get_channel_id_with_timeout(integration, name, 10)
+
+
+def get_channel_id_with_timeout(integration, name, timeout):
```

Comment:
nit: Is there a reason to use a return tuple instead of a dict? It might make its implementation more readable to have named keys returned and used in the task.

Reply:
@MeredithAnya ^